app/controllers/users_controller.rb

```diff
@@ -60,10 +60,11 @@ def update
     @partner_organisations = partner_organisations
 
     reset_mfa = user_params.delete(:reset_mfa)
-    @user.assign_attributes(user_params.except(:reset_mfa))
+    active = user_params[:active] === "true"
```

Comment:
@mec I don't _think_ this needs to be triple equals, does it?

Reply:
No it really shouldn't be! 🙏